bugalink-backend/users/views.py

```diff
@@ -201,3 +216,27 @@ class UserRatingView(viewsets.GenericViewSet, mixins.RetrieveModelMixin):
 
     def get(self, request, *args, **kwargs):
         return self.retrieve(request, *args, **kwargs)
+    
+# GET /users/<user_id>/trip-requests/count?request_status=XXX&trip_status=XXX
```

Comment:
The query params are actually in camelCase, not snake_case, so this is confusing and should be renamed to `requestStatus` and `tripStatus` as those are the query params that work